src/_gettsim/parameters/kindergeld.yaml

```
@@ -171,6 +171,14 @@ kindergeld:
     reference: Art. 6 G. v. 08.12.2022 BGBl. I S. 2230.
     note: Inflationsausgleichsgesetz
     scalar: 250
+  2025-01-01:
+    scalar: 255
+    note: 2. Jahressteuergesetz 2024
```

```suggestion
    note: Steuerfortentwicklungsgesetz (SteFeG)
```
also other instances...